### DISCUSSION TITLE
ADE e_stream_error during download

### DISCUSSION THREAD

**Discussion by michaelbr51** [apprenticeharper/DeDRM_tools#1754] [Q&A] [answered]
Has anyone seen this error before? Never could make it work on my new minipc, re-installed it at least ten times, still the same error. I'm using ADE 2.0.1, under Windows 8.1 guest in virt-manager, under MX Linux 19.4. My questions:
1) Does ADE has limitation in computer authorization? I have my authorization in similar configuration in a laptop, and it's working fine. Do I need to de-authorize my laptop in order to authorize in my minipc?
2) Can DeDRM work with other version of ADE? My thinking is maybe with newer version of ADE, Adobe made older version "unusable"?
![ADE_error](https://user-images.githubusercontent.com/20541206/123681060-9f320600-d849-11eb-9d57-d397da71f4e0.png)

**Comment by michaelbr51** [comment, marked as answer]
Never mind, my mistake was I have 2 different ADE 2.0 exe, one 2.0.1 and the other 2.0.6, the last one was given the error, replaced by the previous one, it's working fine.

**Reply by ElleKayEm** [reply]
Glad you figured it out.  Thanks for reporting back.
